index.rb

```diff
@@ -11,7 +12,33 @@
   gem 'nokogiri'
 end
 
-Show = Struct.new(:time, :link, :title, :description, :venue, :keyword_init => true)
+Show = Struct.new(:time, :link, :title, :description, :venue, :keyword_init => true) do
+  def ical_link
+    'data:text/calendar;base64,' +
+    Base64.urlsafe_encode64(
```

Comment:
Had to change this to `strict_encode64`: https://github.com/davishmcclurg/shows/commit/29bcd5b3b4d72e9226189793728378b95ba39fcd